The header consistency error [a] looks related to this PR
[a]
```
src/RecoTracker/MkFitCore/interface/SteeringParams.h:97:5: error: 'track_score_func' does not name a type
   97 |     track_score_func m_track_scorer;
      |     ^~~~~~~~~~~~~~~~
```

--- a/RecoTracker/MkFitCore/interface/SteeringParams.h
+++ b/RecoTracker/MkFitCore/interface/SteeringParams.h
@@ -1,6 +1,8 @@
 #ifndef RecoTracker_MkFitCore_interface_SteeringParams_h
 #define RecoTracker_MkFitCore_interface_SteeringParams_h
 
+#include "RecoTracker/MkFitCore/interface/FunctionTypes.h"
+
 #include <vector>
 #include <stdexcept>
 

